src/coreclr/vm/interoputil.cpp

```diff
@@ -3114,7 +3114,7 @@ void IUInvokeDispMethod(
 
     // Validate that the target is valid for the specified type.
     if (!IsComTargetValidForType(pRefClassObj, pTarget))
-        COMPlusThrow(kTargetException, W("RFLCT.Targ_ITargMismatch"));
+        COMPlusThrow(kTargetException, IDS_EE_TARGET_MISMATCH);
```

Comment:
```suggestion
        COMPlusThrow(kTargetException, W("RFLCT_Targ_ITargMismatch"));
```
It would save us from duplicating the managed resource string.

Reply:
There is another similar typo in invokeutil.cpp. `RFLCT.Targ_StatFldReqTarg` should be `RFLCT_Targ_StatFldReqTarg`